jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md

```diff
@@ -258,9 +258,8 @@ I cleared out all build artefacts (such as the `*.o` files for D and the `nimcac
 
 It turns out that the DMD compiler is much quicker than the LDC compiler. Obviously all the optimisations that LLVM makes take a little while longer to complete.
 
-
```

Comment:
Extra line was needed for formatting reasons on my blog, can be removed here, so this is fine.